code/modules/clothing/spacesuits/rig.dm

```diff
@@ -101,6 +151,7 @@
 	species_fit = list("Vox")
 	_color = "syndi"
 	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 35, bio = 100, rad = 60)
+	action_button_name = "Toggle Helmet Camera" //This helmet does not have a light, but we'll do as if

```

Comment:
The action button is intended to appear here, since it toggles the Syndie helmet camera